### PR TITLE
[ci] Add sccache to remaining Docker images

### PR DESCRIPTION
**Pull request by driazati**
These just need sccache installed and available on the `PATH` to start using it in CI

cc @areusch

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -145,7 +145,7 @@ def should_skip_ci(pr_number) {
     script: "./tests/scripts/git_skip_ci_globs.py",
     label: 'Check if CI should be skipped due to changed files',
   )
-  if (glob_skip_ci_code == 0) {
+  if (glob_skip_ci_code == 0 && !rebuild_docker_images) {
```
Comment (areusch) [inline Jenkinsfile:148]: should we add this to PR description?

File: docker/install/ubuntu_install_rust.sh [hunk 2]
```diff
@@ -20,8 +20,17 @@ set -euxo pipefail
 
 export RUSTUP_HOME=/opt/rust
 export CARGO_HOME=/opt/rust
+
 # this rustc is one supported by the installed version of rust-sgx-sdk
-curl -s -S -L https://sh.rustup.rs -sSf | sh -s -- -y --no-modify-path --profile minimal --default-toolchain stable
+HOST_ARG=
+if [ "$(getconf LONG_BIT)" == "32" ]; then
```
Comment (areusch) [inline docker/install/ubuntu_install_rust.sh:26]: why this change?

Comment (driazati) [reply docker/install/ubuntu_install_rust.sh:26]: see comment below, 32bit linux needs some prodding to install the right thing

**Review by areusch — APPROVED** [review]
